tests/testthat/test-frames.R

```diff
@@ -21,7 +21,15 @@ test_that("load_frames", {
     sapply(frames$frames[[14]][[2]]$screen$lines, function(x) x[[1]][[1]])
   )
 
-  mockery::stub(load_frames, "system.file", "")
+  local_mocked_bindings(
+    system.file = function(..., package) {
+      if (..1 == "load-cast.js") {
+        ""
+      } else {
+        base::system.file(..., package = package)
+      }
+    }
+  )
```

Comment:
Why is this more complicated than the original one?

Reply:
because `system.file()` is also being called by `load_svg_term()`: https://github.com/r-lib/asciicast/blob/main/R/frames.R#L7 and was getting an error there with the simple `local_mocked_bindings(system.file = function(...) ")`

Reply:
I've tried it again, and the simpler version doesn't work since `local_mocked_bindings()` doesn't bind to the scope of the function like `mockery::stub()` did, hence the distinction.

Reply:
This shows why the mockery approach is better. Now I might need to update the tests every time I call one of the mocked functions. :(

Reply:
The only work around for that is calling a wrapper function, like mentioned in the docs: https://testthat.r-lib.org/reference/local_mocked_bindings.html?q=local_m#namespaced-calls. But IMO thas not a good solution.

Reply:
Yes, if you want to be sure that you are safe, you should only mock functions that are only called once from your package. But that does not seem like a great solution, either.